CoreAlgorithmPlugins/src/au/gov/asd/tac/constellation/plugins/algorithms/clustering/infomap/infomap/InfomapBase.java

```diff
@@ -69,6 +69,13 @@
     protected StringBuilder bestIntermediateStatistics;
 
     private final ArrayList<NodeBase> nonLeafActiveNetwork;
+    
+    private static final String NINE_FORMAT1 = "%.9f, ";
```

Comment:
It feels excessive turning number formatters into constants...

Reply:
I'm not going to completely disagree with you there in that does look a little silly. It is worth noting though that in a number of instances a formatter is being used quite frequently (at least 3 for all the ones that were flagged but a lot more in some cases). For that reason, as silly as it looks, it probably wouldn't hurt to have a constant for them